config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -413,7 +413,7 @@ presubmits:
         - --
         - --build=quick
         - --cluster=
-        - --env=KUBE_FEATURE_GATES=AllAlpha=true,CSIMigration=false,InTreePluginGCEUnregister=false
+        - --env=KUBE_FEATURE_GATES=AllAlpha=true,CSIMigration=false,InTreePluginGCEUnregister=false,DisableCloudProviders=false
```

Comment:
/cc @jpbetz @cici37 
Not quite the build cloud provider flag which should never be disabled for this repo that I warned about. However this is a similar control but this one is a feature gate flag. This is another one we need to be aware of, although with finer grained controls, we may find this one useful for testing KAS+KCM+Kubelet.

Reply:
the alpha job is broken since this merged with this error:

```
E0514 03:54:23.631068    2633 server.go:218] "Failed to set feature gates from initial flags-based config" err="unrecognized feature gate: DisableCloudProviders"
```

Reply:
@liggitt -- I think we've got a chicken-and-egg.

Looks like https://github.com/kubernetes/kubernetes/pull/100136 needs to merge as well.